Airbnb.Application.Tests/UnitTests/AuthControllerTests.cs

```diff
@@ -0,0 +1,67 @@
+using Airbnb.Application.DTOs;
+using Airbnb.Application.Interfaces;
+using Airbnb.Controllers;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+
+namespace Airbnb.Application.Tests;
+
+public class AuthControllerTests
```

Comment:
remove controller tests. We don't test Presentation layer, only Application layer